build/build.ps1

```diff
@@ -111,8 +111,15 @@ function Build {
   }
 
   $nodeReuse = !$ci
+  $useCodecov = $ci -and $env:CODECOV_TOKEN -and ($configuration -eq 'Debug')
```

Comment:
This should be passed to Build.ps1 as a switch, build.ps1 takes arguments that dictate this sort of the "like --cover".

Reply:
Coverage data for local test runs is handled differently from coverage data used by the CI builds.

1. Rather than report the results to codecov.io, the results need to run through ReportGenerator to produce a local HTML view.
2. The OpenCover step extracts PDBs from binaries, leaving a bunch of .pdb and .ppdb files on the file system which are not cleaned in a normal manner. This is fine for the CI since the entire directory is cleared before each build, but would be a nuisance for persistent directories such as the ones on developer machines.

I was hoping to resolve these items separately once we have a chance to review the overall coverage functionality and have a discussion about the additional features we want to see available.